_posts/2017-05-18-install-party.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: "Install Party + Social"
+slug: install-party
+text: 'tomesh monthly install party and social'
+location: Semaphore Studio 307, Claude T. Bissell, 140 St. George St  
+locationLink: http://osm.org/go/ZX6Bw~WNh--?m=
+date: 2017-05-18
```

Comment:
Oh I thought it was:

May 04 - install party/check-in
May 18 - virtual
Jun 01 - install party/check-in
Jun 15 - virtual

See: https://github.com/tomeshnet/documents/commit/ba2938db12cc25257f3f0ee685d7f3087f261490#diff-b127bd3696cae846707a960e8bed6a7cR80

node.tech are also bi-weekly, but separate from the above, and the first one will be on May 11:

https://github.com/tomeshnet/documents/commit/ba2938db12cc25257f3f0ee685d7f3087f261490#diff-b127bd3696cae846707a960e8bed6a7cR190